packages/@azure/servicebus/data-plane/lib/core/managementClient.ts

```diff
@@ -422,8 +422,12 @@ export class ManagementClient extends LinkEntity {
       : (lockTokenOrMessage as string);
     try {
       const messageBody: any = {};
+
+      const lockTokenBuffer: Buffer[] = [];
+      lockTokenBuffer.push(string_to_uuid(lockToken));
+
       messageBody[Constants.lockTokens] = types.wrap_array(
-        reorderLockTokens([lockToken]),
+        lockTokenBuffer,
```

Comment:
Instead of creating a new const, you can directly pass `[string_to_uuid(lockToken)]`